lib/WorkflowHicar.groovy

```diff
@@ -11,11 +11,11 @@ class WorkflowHicar {
     // Check and validate parameters
     //
     public static void initialise(params, log) {
-        genomeExistsError(params, log)
 
+        genomeExistsError(params, log)
 
-        if (!params.fasta) {
-            Nextflow.error "Genome fasta file not specified with e.g. '--fasta genome.fa' or via a detectable config file."
+        if (!params.gtf && !params.gff) {
+            Nextflow.error("No GTF or GFF3 annotation specified! The pipeline requires at least one of these files.")
```

Comment:
Do you have `params.gtf` and `params.gff`? I can't see them in `nextflow.config`

Reply:
They are appended the main.nf.